Libraries_API.ipynb

```diff
@@ -713,6 +664,151 @@
     "    -X DELETE"
    ]
   },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### /query/&lt;library_id&gt;\n",
+    "Add or remove documents the library specified by `library_id` (string) is the unique identifier of the library, as specified in the `/libraries` GET response.\n",
+    "\n",
+    "Functions allowed: add query to a library (POST, GET), delete records from a library (POST).\n",
+    "\n",
+    "#### POST\n",
+    "Add or delete a query to/from a given library. Returns the number of documents added or removed along with the added bibcodes.\n",
+    "\n",
+    "Specify that the request is a POST method by adding `-X POST` to your request. You must include a payload, passed via the `-d` flag, in JSON format. Add this header to your request to indicate your payload is in the appropriate format: `-H \"Content-Type: application/json\"`. The payload must include:\n",
+    "  * query: (json) dict of query parameters as specified in 'search'\n",
```

Comment:
"as specified in 'search'" - do you mean in the search notebook? If so, I'd say something more like "dict of query parameters (see the Search notebook for more details)"